Makefile

```diff
@@ -207,6 +207,15 @@ aks-production: production-cluster
 domains:
 	$(eval include global_config/domains.sh)
 
+# make review pr_id=5432 railsc
+# make qa railsc
+.PHONY: railsc
+railsc: get-cluster-credentials
+	$(eval NAMESPACE_SUFFIX=$(shell echo $(AZURE_SUBSCRIPTION) | awk -F '-' '{print $$NF}'))
```

Comment:
Namespaces don't always map to subscriptions (eg preprod). Can we use the value from tfvars instead?
https://github.com/DFE-Digital/access-your-teaching-qualifications/blob/main/terraform/application/config/test.tfvars.json#L3

Reply:
This should map though right? Because preprod is always the test subscription, as is review and test, so they all pull from the same value, whereas prod is a different subscription and namespace. I can change it but just as a sanity check this should work?

Reply:
OK preprod maps. But it doesn't work for review: https://github.com/DFE-Digital/access-your-teaching-qualifications/blob/main/terraform/application/config/review.tfvars.json#L3
Also we might change the namespaces later on. It's safer not to guess, but get the value from the source of truth.